cpp/tests/interop/dlpack_test.cpp

```diff
@@ -481,6 +493,7 @@ TYPED_TEST(DLPackNumericTests, FromDlpackEmpty1D)
   cudf::table_view input(std::vector<cudf::column_view>{});
   unique_managed_tensor tensor(cudf::to_dlpack(input));
 
+  EXPECT_EQ(nullptr, tensor.get());
   // Verify that from_dlpack(to_dlpack(input)) == input
```

Comment:
This comment doesn't seem to match the `EXPECT_THROW` immediately after it. Can we rewrite?
```suggestion
  // Verify that from_dlpack(nullptr) throws a logic error.
```

Reply:
Arguably it's better to just remove this comment now that the validation that the tensor is `nullptr` is in the code?

Reply:
Either way is fine.

Reply:
Removed.